### PR TITLE
Improved readability of the readme

### PR DESCRIPTION
**Pull request by Nils-ChristianIseke**
Improve the readability of the readme. I will make further commits to revise the rest of the readme if my first pull request is accepted.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # ROSCon 2022 workshop on ros2_control
 
 Thank you for your interest in ros2_control.
-This repository is developed with a purpose of supporting workshop on *ros2_control* @ ROSCon2022 in Kyoto, Japan.
-Regardless if you could or could not participate the repository will provide you with detailed instruction on how utilize *ros2_control* framework and explain functionality and purpose of its individual parts.
+This repository was developed with the purpose of supporting workshop on *ros2_control* @ ROSCon2022 in Kyoto, Japan.
```
Comment (bmagyar) [inline README.md:4]:
```suggestion
This repository was developed with the purpose of supporting the workshop on *ros2_control* @ ROSCon2022 in Kyoto, Japan.
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+Whether you participated or not, this repository will provide you with detailed instructions on how to use the *ros2_control* framework, explaining the functionality and purpose of its individual parts.
 
 ### Installing this repository
 
-After cloning the repository, go to your source workspace and execute following command to install all dependencies:
+After cloning the repository, go to your source workspace and execute following commands to import the necessary repositories and to install all dependencies:
```
Comment (bmagyar) [inline README.md:9]:
```suggestion
After cloning the repository, go to your source workspace and execute the following commands to import the necessary repositories and to install all dependencies:
```

File: README.md [hunk 3]
```diff
@@ -15,36 +15,36 @@ rosdep install -y -i --from-paths .
...
 ## What is *ros2_control*
 
-Simply, ros2_control is a control framework for ROS2.
-But actually, it is much more, it is kernel of ROS2 system that controls robots:
- - it abstracts hardware and low-level control for other framework like MoveIt2 and Nav2;
- - it provide resource access management,
- - controls theirs lifecycle and so on.
+In short, ros2_control is a control framework for ROS2.
+But actually, it is much more, it is the kernel of the ROS2 system that controls robots:
+ - it abstracts hardware and low-level control for other framework such as MoveIt2 and Nav2;
```
Comment (bmagyar) [inline README.md:20]:
```suggestion
 - it abstracts hardware and low-level control for other frameworks such as MoveIt2 and Nav2;
```

**Review by bmagyar — APPROVED** [review]
I added a few more missing articles, thanks for the touch ups!!

**Review by destogl — APPROVED** [review]

**Comment by destogl** [comment]
@Nils-ChristianIseke thanks for the contribution! :partying_face: 

Please take care that are doing heavy rebasing working to keep the branches nice an clean, so before you send new PR take care that you start from the proper branch.

**Pull request merged**